src/config.ts

```diff
@@ -28,20 +28,31 @@ export function defineConfig(config: ViteUserConfig = {}): ViteUserConfig {
       mode: 'development',
     },
   })
+  const { test } = config
 
   return defineVitestConfig({
+    ...config,
```

Comment:
Feels quite repetitive. Could you refactor to use instead:
```ts
import { mergeConfig } from 'vite'
```

Reply:
Sure thing! I have refactored the code in 678439a.